src/database/data/nodes.ts

```diff
@@ -351,5 +351,13 @@ export const NODES_CONFIG: { [key in NetworkId]: StaticNodeConfig[] } = {
       service: 'matic',
       url: 'https://rpc-mainnet.matic.network'
     }
+  ],
+  xDAI: [
+    {
+      name: NetworkUtils.makeNodeName('xDAI', 'blockscout'),
+      type: NodeType.RPC,
+      service: 'blockscout',
```

Comment:
Any reason why it's called `blockscout`? Thats the block explorer right? Same goes for the above line